tools/openocd/modm/st_nucleo_h723zg.cfg

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Shouldn't this rather be MPL-v2 or no license at all?

Reply:
yes, this was a copy paste error from my pull request to the open ocd repo.